content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,271 @@ Resumé tab
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
-  :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
+  User`, with the drop-down menus.
+
+.. important::
+   Employees do not also need to be users. Employees do **not** count towards billing, while *Users*
+   **do** count towards billing. If the new employee should also be a user, the user must be
+   created. After the :guilabel:`User` is created and saved, the new user will appear in the
+   :guilabel:`Related User` field.
+
+   After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then click
+   :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`,
+   :guilabel:`EmaiL Address`, and then select the :guilabel:`Company` from the drop-down menu. Click
+   :guilabel:`Save` after the information is entered.
+
+- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
+  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear
+  only for Belgian companies, and will not be visible for other locations. These sections log the
+  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
+  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of :guilabel:`Holiday Attest (year) -
+  Simple Holiday Pay from previous employer to recover in (year)`, for both N and
+  N-1 categories. For the :guilabel:`Holiday Attest (year) - Previous occupation for Double Holiday
+  Pay Recovery in (year)` section, click :guilabel:`Add a line`, and enter the number of
+  :guilabel:`Months`, the :guilabel:`Amount`, and :guilabel:`Occupation Rate`. Repeat for all
+  entries. Click the :guilabel:`🗑️ (trash can)` icon to delete a line.
+- :guilabel:`Application Settings`: Enter the employee's :guilabel:`Hourly Cost` in a $XX.XX format.
+  This is factored in when the employee is working at a :doc:`work center
+  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This value affects the
+  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.
```

Comment:
you state this in $$ what about European companies for example? 

Also does this impact payroll?

Reply:
I removed the $  so it is universal. Also, I do not believe it affects payroll at all- I believe this is solely for valuating products. I am checking with the experts on this to be sure

Reply:
Yup! That's true!